config/activity_labels/recipe_tea.yaml

```diff
@@ -0,0 +1,61 @@
+version: "1"
+title: "Tea"
+labels:
+  # Item:
+  # - id: Integer identification number of the label.
+  # - label: Short human-readable semantic string to succinctly represent the
+  #          label.
+  - id: 0
+    label: "background"
+  - id: 1
+    label: "measure-12oz-water"
+    full_str: "Measure 12 ounces of water in the liquid measuring cup"
+    depends: ["start"]
+  - id: 2
+    label: "pour-water-kettle"
+    full_str: "Pour the water from the liquid measuring cup into the electric kettle"
+    depends: ["measure-12oz-water"]
+  - id: 3
+    label: "tea-bag-in-mug"
+    full_str: "Place tea bag in mug"
+    depends: ["start"]
+  - id: 4
+    label: "thermometer-turn-on"
+    full_str: "Turn on the thermometer"
+    depends: ["pour-water-kettle"]
```

Comment:
I think this should just have the "start" dependency, and the "pour-water-kettle" dependency should be moved to "thermometer-in-water" step.

Reply:
Yes, change to "start"